api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -170,7 +176,10 @@ private void maybeAlertOnCostThresholds(
 
     // this shouldn't happen, but it did (RW-4678)
     // alert if it happens again
-    if (currentCost < previousCost) {
+    if (currentCost < previousCost
+        &&
+        // was logging many false positives
+        !withinCostTolerance(currentCost, previousCost)) {
```

Comment:
Let's see if we can add an audit entry for this, since it's more user-oriented than system- or application-oriented.

Reply:
The purpose of this log is to alert us to bugs in our system, rather than reflecting an actual user condition.  Does that change your judgment of audit vs log?